docs/src/development/transfer-dedicated.md

```diff
@@ -1,149 +1,135 @@
 ---
-title: "Transferring data to and from a Dedicated cluster"
+title: "Transfer data to and from a Dedicated cluster"
 weight: 13
-sidebarTitle: "Syncing to Dedicated"
+sidebarTitle: "Sync to Dedicated"
 ---
 
-## Backing up staging and production files
+Transferring data to and from a Dedicated cluster slightly differs from the grid.
+For advanced command line utilities, you might need granular SSH access to a specific host.
 
-Platform.sh automatically creates a backup of the staging and production instances on a Dedicated cluster every six hours.
-However, those are only useful for a full restore of the environment and can only be done by the Platform.sh team.
-At times, you'll want to make a manual backup yourself.
+## Back up your files
 
-To create a manual ad-hoc backup of all files on the staging or production environment, use the standard `rsync` command.
+Platform.sh automatically creates backups of the Staging and Production environments of a Dedicated cluster every six hours.
+These are only useful to fully restore an environment and are managed by the support team.
+
+You can make a manual, local, backup yourself by downloading data from your environment to your local system with:
 
 ```bash
-rsync -avzP <USERNAME>@<CLUSTER_NAME>.ent.platform.sh:pub/static/ pub/static/
+platform scp --project=<PROJECT_ID> --environment=<ENVIRONMENT> -r remote:<DIRECTORY_TO_SYNCHRONIZE> <LOCAL_DIRECTORY>
 ```
 
-That will copy all files from the `pub/static` directory on the production instance to the `pub/static` directory,
-relative to your local directory where you're running that command.
-
-## Backing up the staging and production database
+That command copies all files from the `<DIRECTORY_TO_SYNCHRONIZE>` directory in the environment you want to backup to your `<LOCAL_DIRECTORY>`. Before running the command, make sure that you don't overwrite local data (or do a backup first).
 
-To backup your database to your local system, you need to get the database credentials to use.
+## Back up your database
 
-First, login to the cluster and run the following command:
+To backup your database, you need to adapt and run the following command on your local computer:
 
 ```bash
-echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
+platform db:dump --gzip --project=<PROJECT_ID> --environment=<ENVIRONMENT> 
 ```
 
-Which should give a JSON output containing something like this:
-
-```json
-"database" : [
-      {
-         "path" : "main",
-         "service" : "mysqldb",
-         "rel" : "mysql",
-         "host" : "database.internal",
-         "ip" : "246.0.80.64",
-         "scheme" : "mysql",
-         "cluster" : "jyu7wavyy6n6q-main-7rqtwti",
-         "username" : "user",
-         "password" : "",
-         "query" : {
-            "is_master" : true
-         },
-         "port" : 3306
-   }
-]
-```
+For more options, see [database export](../add-services/_index.md#exporting-data).
```

Comment:
:lady_beetle: I'm not certain what the link is intended to point to, since there isn't a section like that on the [add services page](https://docs.platform.sh/add-services.html).

Also, the link text doesn't work within the sentence. It is being treated as if it's a page name, rather than a part of the sentence as a whole.